Lib/fontTools/ttLib/tables/otTables.py

```diff
@@ -711,8 +711,9 @@ def preWrite(self, font):
         if mapping is None:
             mapping = self.mapping = {}
 
-        glyphOrder = font.getGlyphOrder()
-        mapping = [mapping[g] for g in glyphOrder]
+        if type(mapping) == dict:
```

Comment:
why did you change this? When it is _not_ a dict? And what about the missing `else` when it is not `type(mapping) == dict`?

I believe that you don't need to change `VarIdxMap` class, since you aren't even using it here, you're using `DeltaSetIndexMap` class which doesn't use GIDs internally and has a list of integers as `self.mapping`

Reply:
Yeah I changed this before I started using `DeltaSetIndexMap`.  I'll revert these changes.